components/shared_code/src/shared_data_model/sources/jenkins.py

```diff
@@ -37,17 +37,19 @@ def jenkins_access_parameters(*args, **kwargs) -> dict[str, Parameter]:
 JENKINS_TOKEN_DOCS = _TMP_DOC  # nosec hardcoded_password_string
 
 ALL_JENKINS_METRICS = [
+    "change_failure_rate",
     "failed_jobs",
     "job_runs_within_time_period",
     "source_up_to_dateness",
     "source_version",
     "unused_jobs",
 ]
 
+_JOB_ENTITY_NAME_NAME = "Job name"
```

Comment:
LOL.

Reply:
the alternative was.. `_JOB_ENTITY_NAME_ATTRIBUTE_NAME` :D